### ISSUE TITLE
[OS-SUPPORT] Add support for FemboyOS :3

### ISSUE THREAD

**Issue by Sugaryyyy** [TheDarkBug/uwufetch#216]
```
(FemboyOS chroot >w<) root:/# cat /etc/lsb-release 
DISTRIB_ID="FemboyOS"
DISTRIB_RELEASE="11.2-systemd"
DISTRIB_CODENAME="Sugaryy_"
DISTRIB_DESCRIPTION="FemboyOS 23.01 Prerelease 4"
```
:3

**Comment by TheDarkBug** [comment]
Since you are the developer of the distro, could you add a `/etc/os-release` file? I added a temporary "fix" for the `lsb-release` file, but it would be better with `os-release` as it is the standard for `systemd`-based distros.

Also, I "copied" the ascii art from your custom neofetch

**Comment by TheDarkBug** [comment]
Nevermind, I needed the ID, which is "lfs" so it won't work, thanks anyways

**Comment by Sugaryyyy** [comment]
Hello, I have changed the ID from `lfs` to `femboyos` to make it work :3

**Comment by TheDarkBug** [comment]
I fixed it in 3444eb5a188593c699158b83d6ec000039016763

I'll push to main in future

**Comment by Sugaryyyy** [comment]
Okay ty :3
